README.md

```diff
@@ -1 +1,90 @@
-# react-todo-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-app
+
+> Todo-app is a simple todo-list app.
+
+![screenshot](./app_screenshot.png)
+This is just a simple calculator user interface that I built using the [React](https://reactjs.org/) framework.
```

Comment:
- [ ] please update the README file as this is a to-do app and not a calculator

Reply:
Opsies, Good observation 🙈